data/coalition/coalition culture conversations.txt

```diff
@@ -233,3 +233,137 @@ mission "Coalition Folklore: Kimek"
 				decline
 
 
+
+mission "Coalition Departing Trouble - A"
+	minor
+	invisible
+	to offer
+		has "Coalition: First Contact: done"
+		random < 2
+	source
+		government "Coalition"
+		not attributes "coalition station"
+	destination "Ring of Friendship"
+	on enter
+		conversation
+			`Your ship narrowly avoids a few small Coalition vessels as you exit <origin>'s atmosphere. Not a second later, while you're still stunned by the near-crash, once again the <ship> is almost hit by incoming ships, this time Heliarch warships, in what looks like an intense chase after the ships from before.`
+			`	After they've all passed, you check your comms and radar, finding some other civilian ships nearby seem to have experienced a similar shock of their own. Some of the ships linger over <origin> for a while, as if unsure of what to do. Others bolt away with engines at full power, leaving the system in a blink, and some eventually decide to land again, either too curious about the situation, or perhaps wanting to avoid any trouble.`
```

Comment:
```suggestion
			`	After they've all passed, you check your comms and radar, finding some other civilian ships nearby that seem to have experienced a similar shock of their own. Some of the ships linger over <origin> for a while, as if unsure of what to do. Others bolt away with engines at full power, leaving the system in a blink, and some eventually decide to land again, either too curious about the situation, or perhaps wanting to avoid any trouble.`
```
The original sentence is fine, but I think this is more understandable.